### ISSUE TITLE
Snuba consumes high cpu

### ISSUE THREAD

**Issue by samir34ml** [getsentry/snuba#5825]
### Self-Hosted Version

24.3.0

### CPU Architecture

x86_x64

### Docker Version

25.0.4

### Docker Compose Version

2.24.7

### Steps to Reproduce

I have installed sentry self hosted on an aws EC2. The only different that I have made is the following.
In the process of creating the EC2, I attached a new HDD disk to the server and mounted it. I used HDD disk to mount 2 volumes sentry-kafka and sentry-postgres. I edited the following file install/create-docker-volumes.sh (attached to the issue).
[create-docker-volumes.txt](https://github.com/getsentry/self-hosted/files/14665978/create-docker-volumes.txt)
So, now I can mount all the docker volumes data and storage of both sentry-kafka and sentry-postgres to the HDD disk.

The reason why I did this is to save some money and use huge disk of HDD instead of SSD because these volumes consume huge amount of disk space.

### Expected Result

normal behavior and events receiving and issue monitoring.

### Actual Result

no events no issues high cpu utilization even after uninstalling sentry completely and installing it again.

It worked for a while around day or day and half, then sentry stops to report. No new events or issues or transactions.
I had to reset it and re-install it from the beginning but now even before adding any project cpu is very high above 90%. 
![image](https://github.com/getsentry/self-hosted/assets/134782004/49c07718-b535-4530-8058-f026a028956b)

![image](https://github.com/getsentry/self-hosted/assets/134782004/958a1784-92c3-4155-80cc-36fbcd7a43b9)

I tried to add some logs from the docker compose logs command output.
[logs.txt](https://github.com/getsentry/self-hosted/files/14666617/logs.txt)

So any idea regrading what to do next?

Thanks,

**Comment by azaslavsky** [comment]
Moved this over to the snuba repo for more domain-specific triage.

**Comment by FrankHeijden** [comment]
Related: https://github.com/getsentry/self-hosted/issues/2931#issuecomment-2084957867

**Comment by mcannizz** [comment]
@rihards-simanovics can you please tell us more about:
1. What kind of hardware are you using? 
2. How many events / second are you sending to Sentry?

The logged errors you posted may prevent some features from working fully, but I don't think they would cause high CPU usage.

**Comment by rihards-simanovics** [comment]
TL;DR: Long story short updating VPS for better specs has helped to fix the issue.

@mcannizz, I completely reinstalled the app on a better VPS, and everything is stable. I don't think I've implemented everything fully, and typically would only add features as I need them, so that could be the reason for the error logs.

**Comment by mcannizz** [comment]
@rihards-simanovics glad to hear that everything is stable now. I'm going to close this issue, but feel free to re-open if we can help further.
